blockchain/vm/instructions.go

```diff
@@ -907,6 +907,22 @@ func opPush1(pc *uint64, evm *EVM, scope *ScopeContext) ([]byte, error) {
 	return nil, nil
 }
 
+func opSelfdestruct6780(pc *uint64, evm *EVM, scope *ScopeContext) ([]byte, error) {
+	if evm.interpreter.readOnly {
+		return nil, ErrWriteProtection
+	}
+	beneficiary := scope.Stack.pop()
+	balance := evm.StateDB.GetBalance(scope.Contract.Address())
+	evm.StateDB.SubBalance(scope.Contract.Address(), balance)
+	evm.StateDB.AddBalance(common.BigToAddress(beneficiary), balance)
+	evm.StateDB.SelfDestruct6780(scope.Contract.Address())
+	// if tracer := interpreter.evm.Config.Tracer; tracer != nil {
+	// 	tracer.CaptureEnter(SELFDESTRUCT, scope.Contract.Address(), common.BigToAddress(beneficiary), []byte{}, 0, balance)
+	// 	tracer.CaptureExit([]byte{}, 0, nil)
+	// }
```

Comment:
Any issue with this?

Reply:
We klaytn have not applied [call frame tracing](https://github.com/ethereum/go-ethereum/commit/401354976bb44f0ad4455ca1e0b5c0dc31d9a5f5) yet.

Reply:
In the future, we may need to implement those tracers for the `createAccessList` API.

Reply:
Thank you. Could you then add TODO annotation for this?